blocks/columns/columns.css

```diff
@@ -19,7 +19,8 @@
   display: block;
 }
 
-@media (width >= 900px) {
+@media (min-width: 48rem) {
+  /* !! min-width MQ needs refactor */
```

Comment:
I figure we can refactor these when we build the components. I used the bp closest to 900px for now.